### PR TITLE
WEBAPP-119: IDE configuration

### PR DESCRIPTION
**Pull request by maxammann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,12 +85,25 @@ Used Redux extensions:
...
+
+Install the following plugins:
+* PostCSS
+* EJS
+* ESLint
+* React CSS Modules
+
+Go to:
+* Run > Edit Configurations > Defaults > Jest  
+   to set **Configuration file** to **just.config.json**
```
Comment (klinzo) [inline README.md]: should be jest.config.json, not just.config.json

File: .idea/codeStyles/Project.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (klinzo) [inline .idea/codeStyles/Project.xml]: .idea/codeStyles/Project.xml looks very similar to .idea/codeStyleSettings.xml, do we need both respictively duplicated code twice?

File: README.md (hunk 1 above)
Comment (Schedulaar) [reply README.md]: yup

File: .idea/codeStyles/Project.xml (hunk 2 above)
Comment (maxammann) [reply .idea/codeStyles/Project.xml]: good spot! I think this is due to the intellij 2017.3. Which changed the format of this.
I removed the one from the old version. So better update :)

**Comment by maxammann** [comment]
@Schedulaar let me know when you tested this. You should use intellij 2017.3 though.

**Comment by maxammann** [comment]
be careful when changing branches, it will delete you intellij project config and will leave intellij in an unusable state. Merge in WEBAPP-119 or reimport the project.! 🔥

File: README.md [hunk 3]
```diff
@@ -85,12 +85,25 @@ Used Redux extensions:
 You can run the linter by calling **yarn run lint**. Some issues can be fixed automatically by running **yarn run lintfix**
 
 ## IDE
-Just import this project (from existing sources). Run **yarn** and right-click on package.json to show the npm scripts. 
-Also mark the *src* and *public* folder as *Source directory*. In IntelliJ no other plugins are needed. 
+**Make sure you have at least [nodejs 6](https://nodejs.org/) installed**
+
+We suggest IntellJ IDEA Ultimate as IDE. Just import this project (from existing sources).
+Run **yarn** in Terminal and right-click on package.json to show the npm scripts. 
+Also mark the *src* and *public* folder as *Source directory*.
```
Comment (klinzo) [inline README.md]: i think we renamed the "public" to "www", and at least in intelliJ version 2017.3 the option is called mark as "Sources Root'

**Comment by klinzo** [comment]
@maxammann since I haven't read your warning regarding merging WEBAPP-119, I just had the opportunity to completely setting up the project again, so I can confirm your guide is working :) except from the points I have mentioned. Maybe next time post a message to our slack #web-app channel :D

File: README.md (hunk 3 above)
Comment (maxammann) [reply README.md]: done

**Pull request merged**